pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -628,6 +631,51 @@ type (
 	}
 )
 
+// Git groups all configuration options related to enabling GitOps on a
+// cluster and linking it to a Git repository.
+type Git struct {
+	Repo *Repo `json:"repo,omitempty"`
+	// +optional
+	Operator *Operator `json:"operator,omitempty"`
+	// +optional
+	Profiles []*Profile `json:"profiles,omitempty"` // one or many profiles to enable on this cluster
+}
+
+// Repo groups all configuration options related to a Git repository used for
+// GitOps.
+type Repo struct {
+	URL string `json:"url,omitempty"` // the Git SSH URL to the repository which will contain the cluster configuration, e.g. git@github.com:org/repo
+	// +optional
+	Branch string `json:"branch,omitempty"` // the branch under which cluster configuration files will be committed & pushed, e.g. master
+	// +optional
+	Paths []string `json:"paths,omitempty"` // the paths to the directories under which cluster configuration files will be written, e.g. ["kube-system", "base"]
```

Comment:
I would explain this as "the paths to the directories that will be scanned to look for manifests, e.g...."

Reply:
Fixed.